gsa/src/web/pages/nvts/details.js

```diff
@@ -84,6 +93,10 @@ const NvtDetails = ({entity, links = true}) => {
                 {na(getTranslatableSeverityOrigin(severityOrigin))}
               </TableData>
             </TableRow>
+            <TableRow>
+              <TableData>{_('CVSS Date')}</TableData>
+              <TableData>{dateTimeWithTimeZone(severityDate)}</TableData>
+            </TableRow>
```

Comment:
I am not sure if we should display an empty CVSS Date row if no date is available.

Reply:
```suggestion
            <TableRow>
              <TableData>{_('CVSS Date')}</TableData>
              <TableData>{na(dateTimeWithTimeZone(severityDate))}</TableData>
            </TableRow>
```